### PR TITLE
Update runtime to 23.08

### PR DESCRIPTION
**Pull request by kowalski7cc**

### CODE REVIEW

**Comment by AsciiWolf** [comment]
Thanks. I will retest, but I believe that this is still blocked by #56.

**Comment by AsciiWolf** [comment]
Not sure what happened, but it seems everything works fine now. No missing textures. Nice! :-)

**Comment by AsciiWolf** [comment]
@xplshn Is this ok to be merged? I have done a quick testing and everything seems to work fine now.

File: io.github.ec_.Quake3e.OpenArena.appdata.xml [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2014 Richard Hughes <richard@hughsie.com> -->
-<application>
+  <component type="desktop-application">
   <id type="desktop">io.github.ec_.Quake3e.OpenArena</id>
```
Comment (AsciiWolf) [inline io.github.ec_.Quake3e.OpenArena.appdata.xml:4]:
```suggestion
<component type="desktop-application">
  <id>io.github.ec_.Quake3e.OpenArena</id>
```

**Review by AsciiWolf — CHANGES_REQUESTED** [review]

**Comment by xplshn** [comment]
Is this ready? Does it build with the latest Quake3e commit?

**Review by xplshn — APPROVED** [review]

**Review by xplshn — APPROVED** [review]
Why was the "type" changed?

**Comment by AsciiWolf** [comment]
> Why was the "type" changed?

Because `desktop` is deprecated.

**Comment by xplshn** [comment]
So what's the blocker now?

**Comment by AsciiWolf** [comment]
There's no blocker anymore. :-)

**Review by AsciiWolf — APPROVED** [review]

**Pull request merged**